packages/components/ng-ovh-onboarding/README.md

```diff
@@ -0,0 +1,39 @@
+# ng-ovh-onboarding
```

Comment:
Please use component that is extracted here : https://github.com/ovh/manager/pull/3846